tsconfig.json

```diff
@@ -13,8 +13,12 @@
     "resolveJsonModule": true,
     "isolatedModules": true,
     "jsx": "preserve",
-    "incremental": true
+    "incremental": true ,
+    "baseUrl": ".",
+    "paths": {
+      "@/components": ["components/index.tsx"]
+    }
   },
-  "include": ["next-env.d.ts", "**/*.ts", "**/*.tsx"],
+  "include": ["next-env.d.ts", "**/*.ts", "**/*.tsx", "components/Header/index.js"],
```

Comment:
This is a catch all for any directories that contain the file `.ts`

So it would work for a file like this
`home/document/file.ts`
and like this
`home/components/file.ts`